### PR TITLE
Add Community Roles File

### PR DESCRIPTION
**Pull request by prithvi1307**
## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: community-roles.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  - [Security team member](#security-team-member)
+  - [Org Admins](#org-admins)
+
+## Roles
+
+Most of the roles defined herein are defined by membership in a certain GitHub organization or team:
+
+- [litmuschaos org](https://github.com/litmuschaos): The organization under which all of litmuschaos's activity on GitHub is captured.
+- [@litmuschaos/core-maintainers](https://github.com/litmuschaos/community/blob/main/CORE-MAINTAINERS): The team comprised of all maintainers of the litmus repo.
+- [@litmuschaos/maintainers](https://github.com/litmuschaos/community/blob/main/project/litmuschaos-project-maintainers.yaml): The team comprised of all maintainers of the various projects in the litmuschaos organization: litmusctl, litmus-ui, litmus-go,litmus-docs, website and community repos, etc.
```
Comment (Saranya-jena) [inline community-roles.md:21]: we can remove litmus-ui as it has been archived

File: community-roles.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+- Maintain open collaboration with Contributors and other Maintainers
+- Ask for help when unsure and step down considerately
+- A good understanding of the code-base (or equivalent that is governed by the repository, e.g. `litmuschaos/community` or `litmuschaos/website`)
+- Willing to take on long-term responsibility for the project (or a specific part of it)
+- Commitment to the project. Specifically:
+
+  This can be evidenced differently and we want to maintain some general flexibility assessing this. Significant and sustained contributions can be both by showing a long-term level of care with a bigger number of smaller contributions or by a smaller set of sizable contributions. To make it somewhat more comparable, here is an example of commitment we would be happy to accept for a maintainer:
+
+  - Participate in discussions, contributions, code and documentation reviews for 3 months or more,
+  - perform reviews for 10 non-trivial pull requests (total),
```
Comment (Saranya-jena) [inline community-roles.md:83]: We can make it uppercase 😅

**Review by Saranya-jena — CHANGES_REQUESTED** [review]

**Review by SarthakJain26 — APPROVED** [review]

**Review by Saranya-jena — APPROVED** [review]

**Comment by SarthakJain26** [comment]
@prithvi1307 can you please reverify the links, https://github.com/litmuschaos/community/blob/main/ORG-ADMINS this one is not working

**Comment by prithvi1307** [comment]
@Saranya-jena @SarthakJain26 have made the changes for now. Used the Maintainers.md file for reference. Org Admins will be updated moving ahead

**Pull request merged**